daras_ai_v2/web_widget_embed.py

```
@@ -1,10 +1,13 @@
+import copy
 from typing import Any, Iterator
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)